### PR TITLE
Add `options` to `boost_function` on `Query.parse`

### PR DESCRIPTION
**Pull request by ggcampinho**
Add `options` to `boost_function` on `Query.parse`

### CODE REVIEW

File: lib/elasticband/query.rb [hunk 1]
```diff
@@ -139,7 +139,9 @@ def parse_boost_function(boost_options)
           filter = join_filters(parse_filters(boost_options[:boost_where]))
           ScoreFunction::Filtered.new(filter, ScoreFunction::BoostFactor.new(1_000))
         else
-          ScoreFunction::ScriptScore.new(boost_options[:boost_function])
+          boost_function, boost_function_params = boost_options[:boost_function]
+          boost_function_params ||= {}
+          ScoreFunction::ScriptScore.new(boost_function, boost_function_params)
```
Comment (vnegrisolo) [inline lib/elasticband/query.rb]: @ggcampinho why don't you use something like that:

``` ruby
boost_function, boost_function_params = boost_options[:boost_function]
ScoreFunction::ScriptScore.new(boost_function, boost_function_params || {})
```

Comment (ggcampinho) [inline lib/elasticband/query.rb]: Can be, saves one line

**Comment by vnegrisolo** [comment]
:+1:

**Comment by ggcampinho** [comment]
@vnegrisolo can you check again? I removed the version

**Pull request merged**